--- /usr/share/catalyst/targets/stage1/stage1-preclean-chroot.sh	2016-10-02 02:51:53.592702063 +0000
+++ /usr/share/catalyst/targets/stage1/stage1-preclean-chroot.sh	2016-10-02 02:54:53.239709205 +0000
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+export ROOT=/tmp/stage1root
 export RUN_DEFAULT_FUNCS="no"
 
 source /tmp/chroot-functions.sh
@@ -8,7 +9,7 @@
 show_debug
 
 # Now, some finishing touches to initialize gcc-config....
-unset ROOT
+#unset ROOT
 
 setup_gcc
 setup_binutils

--- /usr/share/catalyst/targets/support/bootloader-setup.sh	2016-10-02 18:10:24.023893122 +0000
+++ /usr/share/catalyst/targets/support/bootloader-setup.sh	2016-10-02 18:17:45.694910682 +0000
@@ -269,8 +269,8 @@
 			icfg=$1/isolinux/isolinux.cfg
 			kmsg=$1/isolinux/kernels.msg
 			echo "default ${first}" > ${icfg}
-			echo "timeout 150" >> ${icfg}
-			echo "ontimeout localhost" >> ${icfg}
+			echo "timeout 30" >> ${icfg}
+			echo "ontimeout gentoo" >> ${icfg}
 			echo "prompt 1" >> ${icfg}
 			echo "display boot.msg" >> ${icfg}
 			echo "F1 kernels.msg" >> ${icfg}
--- /usr/share/catalyst/targets/support/rc-update.sh	2016-10-17 22:12:59.787563647 +0000
+++ /usr/share/catalyst/targets/support/rc-update.sh	2016-10-17 22:13:51.697563176 +0000
@@ -19,12 +19,12 @@
 	rc-update del consolefont
 	# We need to add this one, unconditionally
 	rc-update add autoconfig default
-	[[ -e /etc/init.d/splash ]] && rc-update add splash default
-	[[ -e /etc/init.d/fbcondecor ]] && rc-update add fbcondecor default
+	#[[ -e /etc/init.d/splash ]] && rc-update add splash default
+	#[[ -e /etc/init.d/fbcondecor ]] && rc-update add fbcondecor default
 	[[ -e /etc/init.d/sysklogd ]] && rc-update add sysklogd default
 	[[ -e /etc/init.d/metalog ]] && rc-update add metalog default
 	[[ -e /etc/init.d/syslog-ng ]] && rc-update add syslog-ng default
-	[[ -e /etc/init.d/fixinittab ]] && rc-update add fixinittab default
+	#[[ -e /etc/init.d/fixinittab ]] && rc-update add fixinittab default
 
 	# Do some livecd_type specific rc-update changes
 	case ${clst_livecd_type} in
