diff -Naur wxPython-src-3.0.2.0.orig/src/stc/scintilla/src/Editor.cxx wxPython-src-3.0.2.0/src/stc/scintilla/src/Editor.cxx
--- wxPython-src-3.0.2.0.orig/src/stc/scintilla/src/Editor.cxx	2013-02-27 20:04:10.000000000 +0000
+++ wxPython-src-3.0.2.0/src/stc/scintilla/src/Editor.cxx	2018-04-16 07:03:28.379135223 +0000
@@ -17,6 +17,8 @@
 #include <algorithm>
 #include <memory>
 
+#include <cmath>
+
 #include "Platform.h"
 
 #include "ILexer.h"
@@ -5841,9 +5843,9 @@
 }
 
 static bool Close(Point pt1, Point pt2) {
-	if (abs(pt1.x - pt2.x) > 3)
+	if (std::abs(pt1.x - pt2.x) > 3)
 		return false;
-	if (abs(pt1.y - pt2.y) > 3)
+	if (std::abs(pt1.y - pt2.y) > 3)
 		return false;
 	return true;
 }
