diff -Naur linux-4.15.15-ck.orig/Makefile linux-4.15.15-ck/Makefile
--- linux-4.15.15-ck.orig/Makefile	2018-04-09 03:04:07.783040098 +0000
+++ linux-4.15.15-ck/Makefile	2018-04-09 03:06:28.047042372 +0000
@@ -17,7 +17,7 @@
 
 CKVERSION = -ck1
 CKNAME = MuQSS Powered
-EXTRAVERSION := $(EXTRAVERSION)$(CKVERSION)
+#EXTRAVERSION := $(EXTRAVERSION)$(CKVERSION)
 
 # o Do not use make's built-in rules and variables
 #   (this increases performance and avoids hard-to-debug behaviour);
