diff -Naur catalyst-3.0.1.orig/catalyst/main.py catalyst-3.0.1/catalyst/main.py
--- catalyst-3.0.1.orig/catalyst/main.py	2017-12-30 18:56:34.000000000 +0000
+++ catalyst-3.0.1/catalyst/main.py	2018-04-08 06:07:30.059026604 +0000
@@ -438,14 +438,14 @@
 
 	# Namespaces aren't supported on *BSDs at the moment. So let's check
 	# whether we're on Linux.
-	if os.uname().sysname in  ["Linux", "linux"]:
-		# Start off by creating unique namespaces to run in.  Would be nice to
-		# use pid & user namespaces, but snakeoil's namespace module has signal
-		# transfer issues (CTRL+C doesn't propagate), and user namespaces need
-		# more work due to Gentoo build process (uses sudo/root/portage).
-		namespaces.simple_unshare(
-			mount=True, uts=True, ipc=True, pid=False, net=False, user=False,
-			hostname='catalyst')
+	#if os.uname().sysname in  ["Linux", "linux"]:
+	#	# Start off by creating unique namespaces to run in.  Would be nice to
+	#	# use pid & user namespaces, but snakeoil's namespace module has signal
+	#	# transfer issues (CTRL+C doesn't propagate), and user namespaces need
+	#	# more work due to Gentoo build process (uses sudo/root/portage).
+	#	namespaces.simple_unshare(
+	#		mount=True, uts=True, ipc=True, pid=False, net=False, user=False,
+	#		hostname='catalyst')
 
 	# everything is setup, so the build is a go
 	try:
