diff -Naur genkernel-3.5.2.5-orig/gen_initramfs.sh genkernel-3.5.2.5/gen_initramfs.sh
--- genkernel-3.5.2.5-orig/gen_initramfs.sh	2017-12-25 07:09:12.000000000 +0000
+++ genkernel-3.5.2.5/gen_initramfs.sh	2018-03-31 01:04:28.000000000 +0000
@@ -77,7 +77,7 @@
 		echo "Adding devices to cpio:"
 		cat "${TEMP}/initramfs-base-temp.devices"
 	fi
-	${KERNEL_OUTPUTDIR}/usr/gen_init_cpio "${TEMP}/initramfs-base-temp.devices" >"${CPIO}" \
+	gen_init_cpio "${TEMP}/initramfs-base-temp.devices" >"${CPIO}" \
 			|| gen_die "Failed to add devices to cpio"
 }
 
