diff -Naur genkernel-3.4.52.4-orig/defaults/initrd.scripts genkernel-3.4.52.4/defaults/initrd.scripts
--- genkernel-3.4.52.4-orig/defaults/initrd.scripts	2016-01-05 23:55:05.000000000 +0000
+++ genkernel-3.4.52.4/defaults/initrd.scripts	2018-03-31 03:28:08.249971889 +0000
@@ -858,7 +858,7 @@
 	then
 		splash 'verbose' > /dev/null &
 		cat /lib/keymaps/keymapList
-		read -t 10 -p '<< Load keymap (Enter for default): ' keymap
+		read -t 3 -p '<< Load keymap (Enter for default): ' keymap
 		case ${keymap} in
 			1|azerty) keymap=azerty ;;
 			2|be) keymap=be ;;
